samples/project-with-buildSrc/buildSrc/src/main/kotlin/extensions.kt

```diff
@@ -0,0 +1,12 @@
+import org.gradle.api.*
+
+/**
+ * Defines an extension property of Project that can be used in the build script of the project (or any subproject)
+ */
+val Project.profile
+    get() = findProperty("profile") ?: "dev"
```

Comment:
According to the [Kotlin Guidlines](https://kotlinlang.org/docs/reference/coding-conventions.html#property-formatting) this should be in one line